### PR TITLE
m: add page

### PR DESCRIPTION
**Pull request by Aracki**
[m-cli](https://github.com/rgcr/m-cli) alias `m` is Swiss Army Knife for macOS.  
I have added 5 simple examples what `m` is able to do.

### CODE REVIEW

File: pages/osx/m.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# m
+
+> Swiss Army Knife for macOS.
+
+- Get the battery status:
+
+`m battery status`
+
+- Turn off the bluetooth:
```
Comment (sbrl) [inline pages/osx/m.md]: Not sure that `the` is needed here.

File: pages/osx/m.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`m bluetooth off`
+
+- List available filesystems for formatting:
```
Comment (sbrl) [inline pages/osx/m.md]:
```suggestion
- List available the filesystems for formatting:
```

Perhaps this reads better?

File: pages/osx/m.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`m disk filesystems`
+
+- Enable Dock's auto hide feature:
+
+`m docker autohide YES`
+
+- Disable firewall:
```
Comment (sbrl) [inline pages/osx/m.md]:
```suggestion
- Disable the firewall:
```

Perhaps this reads better?

File: pages/osx/m.md (hunk 1 above)
Comment (Aracki) [reply pages/osx/m.md]: Applied!

File: pages/osx/m.md (hunk 2 above)
Comment (Aracki) [reply pages/osx/m.md]: No. I'm sure there is no need for "the" here.

Comment (pxgamer) [reply pages/osx/m.md]: I agree, `the` should either be before the `available` (i.e. `List the available...`) or left as is (which I feel makes more sense). :+1:

Comment (Aracki) [reply pages/osx/m.md]: I would definitely leave as it is.

**Pull request merged**